src/v2/Apps/Consign/Routes/SubmissionFlow/UploadPhotos/Components/PhotoThumbnail.tsx

```diff
@@ -175,20 +183,43 @@ const PhotoThumbnailSuccessState: React.FC<PhotoThumbnailStateProps> = ({
 }) => (
   <>
     <Flex alignItems="center">
-      <Box
-        height={[48, 120]}
-        width={[48, 120]}
-        minWidth={[48, 120]}
-        mr={[15, 2]}
-        bg="black10"
-      >
+      <ImageContainer>
         <Image
           src={photoSrc}
           style={{ objectFit: "cover" }}
           height="100%"
           width="100%"
         />
-      </Box>
+      </ImageContainer>
+      <TruncatedLine variant="xs">{photo.name}</TruncatedLine>
+    </Flex>
+    <Flex alignItems="center" justifyContent="space-between">
+      <Text variant="xs">{formatFileSize(photo.size)}</Text>
+      <RemoveButton handleDelete={onDelete} />
+    </Flex>
+  </>
+)
+
+const PhotoThumbnailProcessingState: React.FC<PhotoThumbnailStateProps> = ({
+  onDelete,
+  photo,
+}) => (
+  <>
+    <Flex alignItems="center">
+      <ImageContainer
+        position="relative"
+        display="flex"
+        alignItems="center"
+        justifyContent="center"
+      >
+        <Media at="xs">
+          <Spinner size="small" />
+        </Media>
```

Comment:
Pay attention to the mobile version, we decided to show a spinner instead of a label since there is really no space for the label. Let me know if you have a better idea for it.